python/adelphi/adelphi/store.py

```diff
@@ -54,7 +54,6 @@ def partition(pred, iterable):
         log.info("Excluding system keyspaces " + ",".join((ks.name for ks in failed)))
     return passed
 
-
```

Comment:
Nit: there are two spaces between the various functions in all the Python packages.  We can change that if desired but a one-off change doesn't seem like a good idea.

Reply:
fixed